### PR TITLE
Ignore some builders

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
It follows https://github.com/gtk-rs/gtk/pull/1019. The point being they stop showing up when running `gir -m not_bound`.

Although, there is a little issue on gir side which completely ignores the ignored builders. I'll send a fix there.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
I should sort the ignores though...

**Comment by GuillaumeGomez** [comment]
The ignore will be taken into account when https://github.com/gtk-rs/gir/pull/933 will be merged. I'll update this PR tomorrow.

File: Gir.toml [hunk 1]
```diff
@@ -356,6 +356,28 @@ builders = [
 
 ignore = [
     "Gtk.Activatable",
+    # base class
+    "Gtk.WidgetBuilder",
+    # deprecated
+    "Gtk.ArrowBuilder",
+    # base class
+    "Gtk.MiscBuilder",
+    # base class
```
Comment (sdroege) [inline Gir.toml:365]: We can easily detect this case

```xml
    <class name="Misc"
           c:symbol-prefix="misc"
           c:type="GtkMisc"
           parent="Widget"
           abstract="1"
           glib:type-name="GtkMisc"
           glib:get-type="gtk_misc_get_type"
           glib:type-struct="MiscClass">
```

Note the `abstract=1` here. That could be parsed by gir

File: Gir.toml [hunk 2]
```diff
@@ -356,6 +356,28 @@ builders = [
 
 ignore = [
     "Gtk.Activatable",
+    # base class
+    "Gtk.WidgetBuilder",
+    # deprecated
```
Comment (sdroege) [inline Gir.toml:361]: It's also an abstract base class

File: Gir.toml (hunk 1 above)
Comment (GuillaumeGomez) [reply Gir.toml:365]: Oh interesting! I'll add it in gir then.

**Comment by GuillaumeGomez** [comment]
The removed builders seem to be a soundness bug we had before. They're not supposed to exist.

**Pull request merged**